types/react-native-awesome-card-io/index.d.ts

```diff
@@ -0,0 +1,74 @@
+// Type definitions for react-native-awesome-card-io 0.8
+// Project: https://github.com/Kerumen/react-native-awesome-card-io
+// Definitions by: Onur Var <https://github.com/OnurVar>
+//                 Emlyn Bolton <https://github.com/emlynmac>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from 'react';
+import { StyleProp, ViewStyle } from 'react-native';
+
+export interface CardDetails {
+    cardType: string; // Localized card type.
```

Comment:
Just going to leave this comment here, as I'm going to merge, but if you change all these inline comments to jsdoc-style comments, so
```suggestion
    /**
      * Localized card type.
      */
    cardType: string;
```
the text will show up in quick-info in the editor, which is usually nice/preferred.

Reply:
Hi @weswigham, thanks for the feedback. I'll make the changes in the next release.